tests/test_vectors.py

```diff
@@ -67,11 +68,18 @@
 _FILE_ENCODINGS[_good_file(u'utf16.ion')] = _ENCODING_UTF16_BE
 _FILE_ENCODINGS[_good_file(u'utf32.ion')] = _ENCODING_UTF32_BE
 
+# 32 or 64 bits.
+_PLATFORM_ARCHITECTURE = struct.calcsize('P') * 8
+
 # Set these Decimal limits arbitrarily high because some test vectors require it. If users need decimals this large,
-# they'll have to do this in their code too. CPython may store these in an ssize_t, so the values should remain in the
-# 32-bit range for compatibility with 32-bit platforms.
-getcontext().Emax = 2147483647
-getcontext().Emin = -2147483648
+# they'll have to do this in their code too. Note: these values are equivalent to the MAX_EMAX and MAX_EMIN values
+# for the platform's architecture (32 or 64 bit); the constants were introduced in Python 3.3.
+if _PLATFORM_ARCHITECTURE == 32:
+    getcontext().Emax = 425000000
```

Comment:
Is there a way we can reference the constants directly instead of reproducing the numeric literals? If not, could you add a link to [the newly defined values](https://python.readthedocs.io/en/stable/whatsnew/3.3.html#id2) in the comment?